index.js

```diff
@@ -0,0 +1,44 @@
+const sumAll = (arr) => {
+  let total = 0
+
+  for (let i = 0; i < arr.length; i++) {
+    total += arr[i]
+  }
+
+  return total
+}
+
+const countPosition = (lineup, position) => lineup.filter((lineup) => lineup.position === position).length
+
+const maxGameIds = (arr) => {
+  for (let i = 3; i < arr.length; i++) {
+    if (arr[i] === arr[i - 1] && arr[i] === arr[i - 2] && arr[i] === arr[i - 3]) {
+      return false
+    }
+  }
+
+  return true
+}
+
+const maxTeamIds = (arr) => {
+  for (let i = 2; i < arr.length; i++) {
+    if (arr[i] === arr[i - 1] && arr[i] === arr[i - 2]) {
+      return false
+    }
+  }
+
+  return true
+}
+
+const validateLineup = (lineup) => {
+  const salaries = lineup.map((lineup) => lineup.salary)
+  const gameIds = lineup.map((lineup) => lineup.gameId).sort()
+  const teamIds = lineup.map((lineup) => lineup.teamId).sort()
```

Comment:
Using reduce here to reformat into objects would make the validation much easier and reduce the amount of looping you're doing. Map is a loop over all players. Sort is a loop of the same number of iterations. Then you for loop over it again. That's three loops over the length of lineup. Could be a lot fewer.